packages/@lwc/ssr-compiler/src/__tests__/utils/expected-failures.ts

```diff
@@ -17,20 +17,17 @@ export const expectedFailures = new Set([
     'attribute-namespace/index.js',
     'attribute-style/basic/index.js',
     'attribute-style/dynamic/index.js',
-    'dynamic-slots/index.js',
+    'context-slotted/index.js',
```

Comment:
I think having an equivalent test for light DOM slots and scoped slots would likely have failed already. We were not passing the `instance` into light DOM / scoped slots.